src/commands/database-instances-create.ts

```diff
@@ -22,6 +25,16 @@ export default new Command("database:instances:create <instanceName>")
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   .action(async (instanceName: string, options: any) => {
     const projectId = getProjectId(options);
+    const defaultDatabaseInstance = await getDefaultDatabaseInstance({ project: projectId });
+    if (defaultDatabaseInstance === "") {
+      throw new FirebaseError(
+        `It looks like you haven't created a Realtime Database instance in this project before. Please run ${clc.cyan.bold(
+          "firebase init"
```

Comment:
Any reason that you're not just telling them to run `firebase init database`?

Reply:
Not specifically. I could do this.